doc/conf.py

```diff
@@ -20,5 +20,5 @@
 except KeyError:
     pass
 
-extensions.append('sphinx.ext.autosectionlabel')
-autosectionlabel_prefix_document=True
\ No newline at end of file
+# extensions.append('sphinx.ext.autosectionlabel')
+# autosectionlabel_prefix_document=True
```

Comment:
What does this change (commenting out these two lines) do?

Reply:
Please do not leave commented-out code. It leads to bit rot. If you absolutely must for some reason then provide a code comment explaining what it's all about. But far better to just delete it.

Reply:
Adds a autogenerated label to each section and adds a prefix based on the name of the file to that label, but it doesn't create unique enough labels leading to all of those previous warnings. 1658 down to 19.

Reply:
Removed it.